### DISCUSSION TITLE
does agent has default cache?

### DISCUSSION THREAD

**Discussion by skim0011** [microsoft/autogen#4125] [Q&A]
I'm iterating 1xx times and imitating group chat inside of it. It clearly has memory somewhere because,

1. when I run the whole 1xx round - the 2nd round takes 1/10th of time 
2. when I run 30 rounds - force pause and rerun then I see the 30 result straight away and then the speed slows it down

how do I configure (turn on and off0 the cache? and how does this work?

**Comment by ekzhu** [comment]
Yes, there is a default cache in v0.2 API. You can set `cache_seed = None` inside a config list entry to disable cache.
